src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -91,11 +93,17 @@ export default class DiscussionArea {
     this.#currentTimeMs = timeMs;
     DiscussionArea.#ELEMENT_TIMESTAMP_SPAN.innerText =
         timestampToString(timeMs);
-    const nearestComments = this.getNearestDiscussionComments(timeMs);
-    if (nearestComments.length == 0) {
+
+    // Remove any existing highlights.
```

Comment:
Can you make these 2 methods? unHighlightComment, highlightNearestComments, or something like that?

Making these methods helps encapsulate the work.

Reply:
Done!